lib/ansible/modules/apt.py

```diff
@@ -350,18 +351,28 @@
     autoclean='Del ',
 )
 
-HAS_PYTHON_APT = True
-try:
+# init module names to keep pylint happy
+apt = apt_pkg = None
+
+def do_apt_imports():
+    global apt, apt_pkg
     import apt
     import apt.debfile
     import apt_pkg
+
+
+try:
```

Comment:
This logic might need to be "fancier" around selecting which Python to use like the dnf one is...